Client run issues

When trying to start the client it fails:

option 1: run "npm start"
- this opens a web page on http://localhost:3000 showing a error log "TypeError: Cannot read property 'render' of undefined"

option 2: "npm run build" and then "serve -s build"
this opens a page on http://localhost:5000 which shows UI to signup. When I try to register, this error is shown on the page:
PreSignUp failed with error Error: Cannot find module 'babel-runtime/regenerator'.

What is the correct way of running the client?

--- a/client/package.json
+++ b/client/package.json
@@ -23,7 +23,7 @@
     "react-router": "^4.2.0",
     "react-router-dom": "^4.2.2",
     "react-scripts": "^3.4.0",
-    "react-social-login": "^3.2.1",
+    "react-social-login": "3.2.1",
     "redux": "^3.7.2",
     "redux-thunk": "^2.2.0",
     "semantic-ui-css": "^2.2.12",

